<launch>
    <!-- topic 转 CAN -->
    <node pkg="socketcan_bridge" name="socketcan_bridge_node" type="socketcan_bridge_node" output="screen">
        <param name="can_device" value="can0" />
    </node>
    <!-- CAN 转 topic -->

    <!-- 示例节点 -->
    
    <node  pkg="socketcan_bridge" name="pub" type="pub" output="screen" />
    <node  pkg="socketcan_bridge" name="sub" type="sub" output="screen" />
</launch>
